pkg/ccl/streamingccl/streamingest/ingest_span_configs.go

```diff
@@ -48,12 +49,12 @@ import (
 // applies to) never overlaps with any other span configuration target. Else,
 // C2C would break the span config reconciliation system.
 //
-// TODO(msbutler): on an initial scan, we need to buffer up all updates and
-// write them to the span config table in one transaction, along with a delete
-// over the whole tenant key span. Since C2C does not lay a PTS on the source
-// side span config table, the initial scan on resumption may miss updates;
-// therefore, the only way to cleanly update the destination side span config
-// table is to write the latest state of the source table and delete all
+// During the rangefeed initial scan, the spanConfigIngestor buffers up all
+// updates and writes them to the span config table in one transaction, along
+// with a delete over the whole tenant key span. Since C2C does not lay a PTS on
+// the source side span config table, the initial scan on resumption may miss
+// updates; therefore, the only way to cleanly update the destination side span
+// config table is to write the latest state of the source table and delete all
 // existing state, in one transaction.
```

Comment:
I might phrase this differently.  The _initial scan_ isn't missing any updates. Rather, something like "The span configuration ingestion does not create a PTS record for the source side span configuration table to avoid the possibility of an errant physical replication job from impacting a system table's GC.  As a result, on resumption we must do a new initial scan, rebuilding the config from scratch, to avoid missing data that may have been GC'd."